craft/base/crafts.lua

```diff
@@ -358,6 +381,36 @@ function Craft:loadDialog(dialog, user)
             end
         end
     end
+    
+    
+    local repairPositions = {12, 13, 14, 15, 16, 17} -- all belt positions
+    local showRepairGroup = false
+    local repairSkill = self:getRepairSkill(user)
+    for k = 1, #repairPositions do
+        local itemAtCharacter = user:getItemAt( repairPositions[k] )
+        local itemStats = world:getItemStatsFromId(itemAtCharacter.id)
+        if itemStats.MaxStack == 1 then
+            for i = 1, #self.products do
+                local product = self.products[i]
+                local productRequirement = product.difficulty
+                if product.item == itemAtCharacter.id and productRequirement <= repairSkill then
+                    if showRepairGroup == false then
+                        dialog:addGroup(user:getPlayerLanguage() == Player.german and "Reparieren" or "Repair")
```

Comment:
Not an issue at all, but pretty sure a function is defined in common.lua that can be used to select text based on player language. This is fine, it just isn't necessary to re-invent.